backends/bmv2/mapAnnotations.h

```diff
@@ -25,31 +25,18 @@ limitations under the License.
 namespace BMV2 {
 
 /**
- * Copy Annotations from architecture block to the corresponding
- * instance in user program.
+ * Build a map from architecture block to its declaration in uer program.
```

Comment:
I am hoping that this pass will not be necessary; annotations are a brittle way to do what I think you are doing.
The architecture description language is rich enough to expose all the information you need, you are building another language layer on top of it with annotations, which is not as precise.